lib/clusters/deviceTemperature.js

```diff
@@ -2,7 +2,17 @@
 
 const Cluster = require('../Cluster');
 
-const ATTRIBUTES = {};
+const ATTRIBUTES = {
+  currentTemperature: { id: 0, type: ZCLDataTypes.int16 },
+  minTempExperienced: { id: 1, type: ZCLDataTypes.int16 },
+  maxTempExperienced: { id: 2, type: ZCLDataTypes.int16 },
+  overTempTotalDwell: { id: 3, type: ZCLDataTypes.uint16 },
+  DeviceTempAlarmMask: { id: 16, type: ZCLDataTypes.map8( 'deviceTemperatureTooLow', 'deviceTemperatureTooHigh') },
+  LowTempThreshold: { id: 3, type: ZCLDataTypes.int16 },
+  HighTempThreshold: { id: 3, type: ZCLDataTypes.int16 },
+  LowTempDwellTripPoint: { id: 3, type: ZCLDataTypes.uint24 },
+  HighTempDwellTripPoint: { id: 3, type: ZCLDataTypes.uint24 },
```

Comment:
Lets keep the casing consistent. By the way, eslint would have notified you of this!